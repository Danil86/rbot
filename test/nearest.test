do move to entity nearest reachable object * then jump done