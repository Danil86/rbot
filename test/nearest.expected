["taskList",[[["move to",["entity nearest reachable object *"]],["jump",[]]]]]
